src/trix/models/html_parser.js

```diff
@@ -386,6 +386,15 @@ export default class HTMLParser extends BasicObject {
     return tagName(element) === "br" && this.isBlockElement(element.parentNode) && element.parentNode.lastChild === element
   }
 
+  needsTableSeparator(element) {
+    if (config.theme.removeEmptyTableCells) {
```

Comment:
What do you think of changing `removeEmptyTableCells` to `removeBlankTableCells`? Since it will remove cells with whitespace too.